### PR TITLE
revert insideSVG check for pointer interaction

### PR DESCRIPTION
**Pull request by ztsai**
revert #2416(#2821) to unblock slate

before 2416: http://jsfiddle.net/crjz5531/8/
after 2416: http://jsfiddle.net/crjz5531/9/
now: http://jsfiddle.net/crjz5531/10/

### CODE REVIEW

File: test/interactions/pointerInteractionTests.ts [hunk 1]
```diff
@@ -195,7 +195,7 @@ describe("Interactions", () => {
 
         assert.isFalse(enterCallback1Called, "callback 1 was disconnected from pointer enter interaction");
         assert.isTrue(enterCallback2Called, "callback 2 is still connected to the pointer enter interaction");
-
+        pointerInteraction.offPointerEnter(enterCallback2);
```
Comment (bluong) [inline test/interactions/pointerInteractionTests.ts]: Why this addition

File: test/interactions/pointerInteractionTests.ts [hunk 2]
```diff
@@ -218,9 +218,10 @@ describe("Interactions", () => {
         overlay = TestMethods.getSVGParent().append("div").style({
           height: SVG_HEIGHT + "px",
           width: SVG_WIDTH + "px",
-          position: "absolute",
-          top: SVG_HEIGHT / 2 + "px",
-          left: SVG_WIDTH / 2 + "px"
+          position: "relative",
+          top: -SVG_HEIGHT / 2 + "px",
+          left: SVG_WIDTH / 2 + "px",
+          background: "black"
```
Comment (bluong) [inline test/interactions/pointerInteractionTests.ts]: Why these changes?

File: test/interactions/pointerInteractionTests.ts (hunk 1 above)
Comment (ztsai) [inline test/interactions/pointerInteractionTests.ts]: for test isolation

File: test/interactions/pointerInteractionTests.ts (hunk 2 above)
Comment (ztsai) [inline test/interactions/pointerInteractionTests.ts]: `absolute` -> `relative` because `absolute` was wrong overlay should be relative to svg
`background color` is for debugging purpose, it shows where the overlay is

File: test/interactions/pointerInteractionTests.ts (hunk 1 above)
Comment (bluong) [inline test/interactions/pointerInteractionTests.ts]: A new interaction is created every single time in the `beforeEach` which exists to help the isolation

**Comment by bluong** [comment]
build break.
